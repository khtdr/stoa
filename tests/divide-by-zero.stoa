#!/usr/bin/env stoa

print 1 / 0;
