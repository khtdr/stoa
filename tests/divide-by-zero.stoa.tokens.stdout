PRINT[3,1]
NUMBER[3,7]
SLASH[3,9]
NUMBER[3,11]
SEMICOLON[3,12]
